Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+pipeline {
+  agent any
```

Comment:
So we only have linux agents, right? No MacOS from here?

Reply:
OSX machines are labelled mac. https://github.com/mavlink/qgroundcontrol/blob/master/Jenkinsfile#L55

Docker slaves are labelled docker.